src/ya-comiste-react-native/package.json

```diff
@@ -28,19 +28,19 @@
     "react": "16.13.1",
     "react-native": "0.63.4",
     "react-native-navigation": "^7.6.0",
-    "react-relay": "0.0.0-master-fa1c75d2",
-    "relay-runtime": "0.0.0-master-fa1c75d2"
+    "react-relay": "0.0.0-master-c02d9722",
```

Comment:
Would it be possible to use the experimental here maybe? We could start playing with hooks?

Reply:
@tbergq I think it would not because experimental Relay depends on unstable React which is not compatible with React Native requirements (it has pinned peer dependency on React 16.13.1 exactly).